shared/types/equipment.types.ts

```diff
@@ -1,14 +1,20 @@
-export enum EquipmentTypes {
-    WEAPON   = 'weapon',
-    SHIELD   = 'shield',
-    NECK     = 'neck',
-    WAIST    = 'waist',
-    HEAD     = 'head',
-    CHEST    = 'chest',
-    ARMS     = 'arms',
-    LEGS     = 'legs',
-    FEET     = 'feet',
-    RING     = 'ring',
-    EARRING  = 'earring',
-    TALISMAN = 'talisman'
+export enum ArmorTypes {
+    HEAD  = 'head',
+    UPPER = 'upper',
+    LOWER = 'lower'
+}
+
+export enum WeaponTypes {
+    DAGGER       = 'dagger',
+    AXE          = 'axe',
+    HAMMER       = 'hammer',
+    SWORD        = 'sword',
+    GREAT_SWORD  = 'great-sword',
+    GREAT_AXE    = 'great-axe',
+    GREAT_HAMMER = 'great-hammer',
+    STAFF        = 'staff',
+    SPEAR        = 'spear',
+    CLUB         = 'club',
+

```

Comment:
Food for thought: 

BOW = 'bow'
DUAL_DAGGER = 'dual-dagger'
DUAL_SWORDS = 'dual-swords' 
FISTS = 'fists'